app/services/bulk_upload/sales/year2024/row_parser.rb

```diff
@@ -514,7 +514,7 @@ def valid?
 
         question = log.form.get_question(error.attribute, log)
 
-        if question.present? && setup_question?(question)
+        if question.present? && setup_question?(question) && error.type != :skip_bu_setup_error
```

Comment:
What if we change the type to `:skip_bu_error` and not add it to the fields at all? That way if there are other errors, they can still be added, and in this specific instance, not displaying the error on the type wouldn't be the worst, cause we wouldn't clear type anyway and it's a compound validation so it would display on all the other relevant fields

Reply:
Ooh, I like that - yeah, I think that solves my concern.

Reply:
Great! Updated it now!